docker/docker-compose-refqa.yml

```diff
@@ -0,0 +1,36 @@
+version: '2.1'
+
+services:
+  openmrs-referenceapplication-mysql:
+    image: mysql:5.6
+    command: "mysqld --character-set-server=utf8 --collation-server=utf8_general_ci"
+    environment:
+      - MYSQL_DATABASE=openmrs
+      - MYSQL_ROOT_PASSWORD=Admin123
+      - MYSQL_USER=openmrs
+      - MYSQL_PASSWORD=Admin123
+    healthcheck:
+      test: "exit 0"
+    ports:
+      - "3306:3306"
+
+  openmrs-referenceapplication:
+    image: openmrs/openmrs-reference-application-distro:qa
+    depends_on:
+      - openmrs-referenceapplication-mysql
+    ports:
+      - "8080:8080"
+    environment:
+      DB_DATABASE: openmrs
+      DB_HOST: openmrs-referenceapplication-mysql
+      DB_USERNAME: openmrs
+      DB_PASSWORD: Admin123
+      DB_CREATE_TABLES: 'true'
+      DB_AUTO_UPDATE: 'false'
+      MODULE_WEB_ADMIN: 'false'
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/openmrs/"]
+      timeout: 20s
+    volumes:
+      - ./csrfguard.properties:/usr/local/tomcat/.OpenMRS/csrfguard.properties
+      - ../omod/target//^fhir2.+.omod$$/:/usr/local/tomcat/.OpenMRS/modules//^fhir2.+.omod$$/
```

Comment:
Hey @ibacher, after following @dkayiwa's suggestion, it is like using Pattern Matching does seam to work. 
Any objections

Reply:
> Does the 1.4.0-SNAPSHOT get automatically deleted?

Logically, yeah.  `/^fhir2.+.omod$$/` matches any file starting with `fhir2.` and ends with `.omod`

Reply:
I thought this just copies over and preserves the file, but has nothing to do with deleting it.

Reply:
i using the knowledge i got from` - ./csrfguard.properties:/usr/local/tomcat/.OpenMRS/csrfguard.properties`, is it that our created `csrfguard.properties` replaces the one in the docker container.

I thought this would work as the same with `- ../omod/target//^fhir2.+.omod$$/:/usr/local/tomcat/.OpenMRS/modules//^fhir2.+.omod$$/`